charts/actions-runner-controller/docs/UPGRADING.md

```diff
@@ -37,7 +37,7 @@ helm repo update
 
 # helm upgrade [RELEASE] [CHART] [flags]
 helm upgrade actions-runner-controller \
-  actions-runner-controller/actions-runner-controller \
+  actions/actions-runner-controller \
```

Comment:
```suggestion
  actions-runner-controller/actions-runner-controller \
```

This should match with the local chart repo name you decide when you run `helm repo add $REPO $URL`, because you run helm upgrade using the `$REPO`, like `helm upgrade actions-runner-controller $REPO/actions-runner-controller`.

I think we are not going to change the local repo name anywhere in the documentation yet, so we can safely keep it as-is in this pull request.